web-common/src/features/entity-management/resource-icon-mapping.ts

```diff
@@ -21,4 +23,22 @@ export const resourceIconMapping = {
   [ResourceKind.Theme]: ThemeIcon,
   [ResourceKind.Report]: ReportIcon,
   [ResourceKind.Alert]: AlertCircleOutline,
+  [ResourceKind.ProjectConfig]: Settings,
+  file: FileIcon,
+};
+
+export const resourceColorMapping = {
```

Comment:
Nice! We should use this in the sidebar too (but we can also do that in a different PR)